src/classes/vehicle.py

```diff
@@ -12,6 +12,8 @@
 SAFETY_DISTANCE_BEHIND_VEHICLE = 6
 TRAFFIC_LIGHT_SAFETY_DISTANCE = 3
 EMERGENCY_DISTANCE = 5
+MAX_ACCELERATION = 3.5
+FPS = 1/60
```

Comment:
this would be frame duration, not fps (frame per second). But as I mentioned we don't need to base it off a frame. It would be fine to call it some sort of interval

Reply:
let's not use delta_time - I think it can be more descriptive. Also there's already a delta time that represents time between each iteration of the simulation loop